apps/teams-test-app/src/components/NestedAppAuthAPIs.tsx

```diff
@@ -152,36 +152,58 @@ const NestedAppAuthAPIs = (): ReactElement => {
       }),
     });
 
-  const AddChildIframeSection = (): React.ReactElement | null => {
+  const AddChildIframeSection = (): React.ReactElement => {
     const [iframeAdded, setIframeAdded] = useState(false);
 
     const addChildIframe = (): void => {
       if (iframeAdded) {
-        console.log('Iframe already added.');
         return;
       }
 
       const iframeContainer = document.getElementById('nestedChildIframeContainer');
       if (!iframeContainer) {
-        console.error('Container not found: nestedChildIframeContainer');
+        console.error('Iframe container not found');
         return;
       }
 
       const childIframe = document.createElement('iframe');
-      childIframe.src = `${window.location.href}?appInitializationTest=true&groupedMode=NestedAppAuthAPIs`;
+      childIframe.src = '/naa_childIframe.html';
       childIframe.id = 'nestedAuthChildIframe';
-      childIframe.width = '100%';
-      childIframe.height = '400px';
+      childIframe.style.width = '100%';
+      childIframe.style.height = '400px';
       childIframe.style.border = 'none';
 
       iframeContainer.appendChild(childIframe);
       setIframeAdded(true);
+
+      // Send payload to the iframe after it loads
+      childIframe.onload = () => {
+        const payloads = {
+          defaultPayloadForBridge: NestedAppAuthRequest,
+          defaultPayloadForTopWindow: JSON.stringify({
+            id: '2',
+            func: 'nestedAppAuth.execute',
+            args: [],
+            data: NestedAppAuthRequest,
```

Comment:
Hmmm generally speaking, all of functions will have an api version tag and yours also break the rule here because you postMessage directly to the iframe using `postMessage` directly. Could you please add a version tag?

Reply:
Okay @baljesingh  says that he would create a ticket for tracking adding the tag. Because if it is not added, telemetry on this API will have `undefined_tag` in our database, which doesn't make sense.

Reply:
As discussed offline, Api version tag need to be added in NAA flow as well. I will handle this later. Filling the bug now.